scania-digital-design-system/tegel
# fix(prefix change): cleanup

**Describe pull-request**  
Cleanup for the prefix change. Some stories needed updated prefix for button and some files needed to be deleted.

**Solving issue**  
Fixes: -

**How to test**  
1. Checkout branch
2. Search for sdds in components folder
3. Check that the result are only for dropdown (which hasn't been merged yet)
